### ISSUE TITLE
Show shortname instead of just a dot.

### ISSUE THREAD

**Issue by cyberorg** [liamcottle/meshtastic-map#39]
Hi, would it be possible to show shortname instead of just a dot?

See this for example: https://meshtastic.taubetele.com/
https://github.com/kkwestt/meshtastic.taubetele.com

Cheers

**Comment by VigibotDev** [comment]
Good idea, this is what I do on my D3.js mesh grapher https://www.serveurperso.com/
It use this project SQL database to feed data
